vault/raft.go

```diff
@@ -509,7 +524,7 @@ func (c *Core) raftTLSRotatePhased(ctx context.Context, logger hclog.Logger, raf
 					continue
 				}
 
-				nextRotationTime = next
+				nextRotationTime = getNextRotationTime(next)
```

Comment:
Can we move the ticker.Reset to here, as well as eliminate the backoff variable and just put all the time calculation logic here?  Feels like it would be safer and clearer.

Reply:
Is your thought to put all of this logic into the current `getNextRotationTime` (maybe with a different name)? That might be tricky as we have to worry about rotation time for the initialization of the ticker, the reset, and the handling of the error returned by the `rotateKeyring` in the case of the backoff.

Reply:
No, I propose to leave getNextRotationTime alone, but get rid of the backoff bool, the `continue`, and just do all the calculations for nextRotationTime here in this `case`.

Reply:
That way we can have more confidence that the padding we add to nextRotationTime is big enough (and possibly can be smaller than 1m), because we'll update the ticker with it immediately.

Reply:
Ah sorry I misunderstood, moved things around a bit.